docs/devguides/contribution_guide.rst

```diff
@@ -0,0 +1,92 @@
+
+Contribution Guide
+=====================
+
+We are happy when you share your research with us and when you improve our
+code! There is an easy way to contribute to our project, follow the steps
+below. Your commit will be reviewed and added to our master branch. Moreover,
+if you are a dedicated contributor, you have a chance to get our t-shirt,
+get invited to one of our events or even join our team ; )
+
+How to contribute:
+
+#. Don't start the coding first. You should **post an issue** to discuss the features
+   you want to add. If our team or other contributors accept your offer
+   or give a +1, assign the issue to yourself. Now proceed with coding : )
+
+#. **Add clear comments** to each line of your code to let other people understand
+   the flow of your mind.
+
+#. **Clone and/or update** your checked out **copy of DeepPavlov** to ensure
+   you have the most recent commits from the master branch:
+
+    .. code:: bash
+
+        git clone 
+        cd 
```

Comment:
Paths are missing for the `git clone` and `cd` commands

Reply:
resolved in 
https://github.com/deepmipt/DeepPavlov/pull/947/commits/d7ba95cb3339db565b3569369c1cdf290cabecd4